week-1/8-advanced-selectors/styles.css

```diff
@@ -1 +1,11 @@
 /* Try different box model properties below */
+.side-header > p {
+  color: white;
+  margin-bottom: 8;
+}
+.site footer > p {
+  text-align: center;
+}
+.message:last-child {
+  box-shadow: 4px 4px 0 #dba944;
+}
```

Comment:
You could also just use `.site-header p` and `.site-footer p` here, which would apply to any paragraphs, not just immediate children. This might be preferable if someone changes the HTML later, for example, and adds another `div` inside the header or footer, but it depends on the situation.